src/database/migrations/create-1-department.js

```diff
@@ -1,26 +1,25 @@
 export default {
-  up: (queryInterface, Sequelize) => queryInterface.sequelize.query('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";')
-    .then(() => queryInterface.createTable('departments', {
-      id: {
-        allowNull: false,
-        primaryKey: true,
-        type: Sequelize.UUID,
-        defaultValue: Sequelize.literal('uuid_generate_v4()')
-      },
-      name: {
-        type: Sequelize.STRING,
-        allowNull: false,
-      },
-      createdAt: {
-        allowNull: false,
-        type: Sequelize.DATE,
-        defaultValue: new Date()
-      },
-      updatedAt: {
-        allowNull: false,
-        type: Sequelize.DATE,
-        defaultValue: new Date()
-      }
-    })),
-  down: (queryInterface, Sequelize) => queryInterface.dropTable('departments')
+  up: (queryInterface, Sequelize) => queryInterface.sequelize.query('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";').then(() => queryInterface.createTable('departments', {
+    id: {
+      allowNull: false,
```

Comment:
nice work @sa-ma , I believe we an do this better and cleaner using 
```javascript
  up: (queryInterface, Sequelize) =>
    queryInterface.createTable('departments', {
      id: {
        allowNull: false,
        primaryKey: true,
        type: Sequelize.STRING,
        defaultValue: Sequelize.UUIDV4,
      }, 
```
if otherwise tell me what you think, regarding this and other models defined using this pattern

Reply:
Eslint will throw an error. If there is no return statement it should be on the same line

Reply:
@sa-ma Really good job, I also love the way you defines your request associations, but can you explain why the request migration has status, accId and reqId if I am right, and the request model only contain *status*

Reply:
That was the change requested in the previous conversations. I was told to migrate all trip related information to trip table. Status is as it says status of a request if the request is pending, accepted or rejected

Reply:
Point taken.

Reply:
> Eslint will throw an error. If there is no return statement it should be on the same line

You can disable the eslint inline